orttraining/orttraining/python/training/ortmodule/ortmodule.py

```diff
@@ -135,3 +160,30 @@ def _replicate_for_data_parallel(self):
 
         raise NotImplementedError("ORTModule is not compatible with torch.nn.DataParallel. "
                                   "Please use torch.nn.parallel.DistributedDataParallel instead.")
+
+    def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
+                                missing_keys, unexpected_keys, error_msgs):
+        """Override original method to delegate execution to the original PyTorch user module"""
+
+        self._module.original_module._load_from_state_dict(state_dict, prefix, local_metadata, strict,
```

Comment:
add a comment explaining why delegation is needed. In the future we can forget and remove it or not remove when we could